cloudflare/src/framework/response/mod.rs

```diff
@@ -9,7 +9,9 @@ use serde_json::value::Value as JsonValue;
 pub struct ApiSuccess<ResultType> {
     pub result: ResultType,
     pub result_info: Option<JsonValue>,
+    #[serde(default)]
```

Comment:
I think it's more correct to make this `Option<JsonValue>`, rather than defaulting to `JsonValue::Null`. What do you think @sssilver -- one downside is that it would be a breaking change (although a very easy to fix one)

Reply:
I agree with @adamchalmers here. The reason I chose to do it this way is that I felt that we could do just a simple patch release with the changes from this and #174 if we kept this nonbreaking.